src/input/split.h

```diff
@@ -0,0 +1,18 @@
+#ifndef MYSPLIT_H
```

Comment:
Do we have a convention for include guards? Should by INPUT_SPLIT_H imo, but I see other files handle it differently as well.

Reply:
did the proposed change